bundles/org.openhab.core.voice/src/main/java/org/openhab/core/voice/VoiceManager.java

```diff
@@ -144,17 +145,18 @@ public interface VoiceManager {
      * @param ks the keyword spotting service to use or null to use the default service
      * @param stt the speech-to-text service to use or null to use the default service
      * @param tts the text-to-speech service to use or null to use the default service
-     * @param hli the human language text interpreter to use or null to use the default service
+     * @param hlis list of human language text interpreters to use, they are executed in order until the first
+     *            successful response, or empty to use the default service
      * @param source the audio source to use or null to use the default source
      * @param sink the audio sink to use or null to use the default sink
-     * @param Locale the locale to use or null to use the default locale
+     * @param locale the locale to use or null to use the default locale
      * @param keyword the keyword to use during keyword spotting or null to use the default keyword
      * @param listeningItem the item to switch ON while listening to a question
      * @throws IllegalStateException if required services are not all available or the provided locale is not supported
      *             by all these services or a dialog is already started for this audio source
      */
     void startDialog(@Nullable KSService ks, @Nullable STTService stt, @Nullable TTSService tts,
-            @Nullable HumanLanguageInterpreter hli, @Nullable AudioSource source, @Nullable AudioSink sink,
+            List<HumanLanguageInterpreter> hlis, @Nullable AudioSource source, @Nullable AudioSink sink,
             @Nullable Locale locale, @Nullable String keyword, @Nullable String listeningItem)
             throws IllegalStateException;
```

Comment:
As I said above: add to API, do not remove from API. This removes the old method from the API and breaks all code USING these interface. As it is exported to scripts, this is bad. Adding new methods only breaks code IMPLEMENTING this interface. Since I believe it is unlikely that there are other implementations, this is less invasive and preferred, therefor keep the old method:

```
        void startDialog(@Nullable KSService ks, @Nullable STTService stt, @Nullable TTSService tts,
             List<HumanLanguageInterpreter> hlis, @Nullable AudioSource source, @Nullable AudioSink sink,
             @Nullable Locale locale, @Nullable String keyword, @Nullable String listeningItem)
             throws IllegalStateException;

        void startDialog(@Nullable KSService ks, @Nullable STTService stt, @Nullable TTSService tts,
             @Nullable HumanLanguageInterpreter hli, @Nullable AudioSource source, @Nullable AudioSink sink,
             @Nullable Locale locale, @Nullable String keyword, @Nullable String listeningItem)
             throws IllegalStateException;

```

The implementation of the old method is easy, something like

```
        void startDialog(@Nullable KSService ks, @Nullable STTService stt, @Nullable TTSService tts,
             @Nullable HumanLanguageInterpreter hli, @Nullable AudioSource source, @Nullable AudioSink sink,
             @Nullable Locale locale, @Nullable String keyword, @Nullable String listeningItem)
             throws IllegalStateException {
            if (hli == null) {
                startDialog(ks, stt, tts, List.of(), source, sink, locale, keyword, listeningItem); 
            } else {
                startDialog(ks, stt, tts, List.of(hli), source, sink, locale, keyword, listeningItem); 
            }
```

If you think that there is no use-case for the old method, you can mark it as `@Deprecated`. But in this case I believe it is better to keep it.

Applies also below for `listenAndAnswer`.

Reply:
Done, sorry for the delay and thank you for the detailed explanation.